_includes/image

```diff
@@ -0,0 +1,14 @@
+{% include base_path %}
```

Comment:
I've deprecated the `base_path` include as Jekyll provides filters to do this now.